marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 5983605072600952,
+            "layout": {
+                "height": 15,
+                "width": 47,
+                "x": 24,
+                "y": 65
+            }
+        },
+        {
+            "definition": {
+                "background_color": "vivid_blue",
+                "content": "Database summary",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 3172903413969796,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 128,
+                "y": 0
+            }
+        },
+        {
+            "definition": {
+                "background_color": "blue",
+                "content": "Requests",
+                "font_size": "14",
+                "show_tick": false,
+                "text_align": "center",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
+            },
+            "id": 6184667112701002,
+            "layout": {
+                "height": 5,
+                "width": 38,
+                "x": 128,
+                "y": 47
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.transactions.total_transactions{*}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "title": "Total number of open transactions",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 6411568736004342,
+            "layout": {
+                "height": 15,
+                "width": 47,
+                "x": 24,
+                "y": 26
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "[TODO] What query_read_rate and total",
```

Comment:
I was still searching for the best thing to say here

Reply:
Got it—maybe we could put something like, `Query read rate and large read rate [indicate](https://docs.marklogic.com/guide/opsdir/Analyse#id_21175) overall disk I/O performance during queries.` This might also be a good opportunity to explain what "Sum of the processing rate factors" means. On the other hand, it's common enough in out-of-the-box dashboards not to include explanatory text that I think we could remove the text here.